bip-0003.md

```diff
@@ -0,0 +1,708 @@
+* Authors — The names (or pseudonyms) and email addresses of all authors of the BIP. The format of each authors header
+  value must be
+
+      Random J. User <address@dom.ain>
+
+  Multiple authors are recorded on separate lines:
+
+      Authors: Random J. User <address@dom.ain>
+               Anata Sample <anata@domain.example>
+
+* Shepherds — Additional owners of the BIP that are not authors. The Shepherds header uses the same format as the
+  Authors header. See the [BIP Ownership](#bip-ownership) section above.
+* Status — The stage of the workflow of the proposal. See the [Workflow](#workflow) section below.
+* Type — See the [Bip types](#bip-types) section below for a description of the three BIP types.
+* License and License-Code — These headers list SPDX License Identifier(s) of the acceptable licenses under which the
+  BIP and corresponding code are available. See the [BIP Licensing](#bip-licensing) section below for a description of
+  the Licenses and their SPDX License Identifiers. If there are multiple acceptable licenses, each should be on a
+  separate line.
+* Discussion — The Discussion header points the audience to relevant discussions of the BIP, e.g., the mailing list
+  thread in which the idea for the BIP was discussed, a thread where a new version of the BIP was presented, or relevant
+  discussion threads on other platforms. Entries take the format "yyyy-mm-dd: URL", e.g., `2009-01-09:
+  https://www.mail-archive.com/cryptography@metzdowd.com/msg10142.html`, using the date and URL of the start of the
+  conversation. Multiple discussions should be listed on separate lines.
+* Version — The current version number of this BIP. See the [Changelog](#changelog) section below.
+* Requires — A list of existing BIPs the new proposal depends on. If multiple BIPs
+  are required, they should be listed in one line separated by a comma and space (e.g., "1, 2").
+* Replaces and Superseded-By — BIPs may have a Replaces header that contains the number of an older BIP that it renders
+  obsolete. In turn, a Superseded-By header indicates that a BIP has been rendered obsolete by the later document with
+  the provided number.
+
+#### Auxiliary Files
+
+BIPs may include auxiliary files such as diagrams and source code. Auxiliary files must be included in a subdirectory
+for that BIP named `bip-XXXX`, where "XXXX" is the BIP number zero-padded to four digits. File names in the subdirectory
+do not need to adhere to a specific convention.
+
+### BIP Types
+
+* A **Specification BIP** defines a set of technical rules affecting the interoperability of implementations. The
+  distinguishing feature of a Specification BIP is that it can be implemented, and implementations can be compliant with
+  it. Specification BIPs must have a Specification section, should have a Backward Compatibility section, and should come with or refer to a reference implementation and test vectors.
```

Comment:
Per line 102, for clarity (also, aren't backward compat sections required, or only encouraged?)

```suggestion
  it. Specification BIPs must have a Specification section and a Backward Compatibility section, and for "Complete" status must include or refer to a reference implementation and test vectors.
```